README.md

```diff
@@ -98,6 +98,91 @@ This will generate the controler for that model, along with Create, Edit, Delete
 
 To create asynchronous jobs implement [Hangfire](https://www.hangfire.io).
 
+## Authentication
+
+#### Identity
+
+In order to add authentication in our application, we use Identity. For this, we must add the Identity Package:
+
+```bash
+    dotnet add package Microsoft.AspNetCore.Identity.EntityFrameworkCore --version 2.0.0
+```
+
+Now we have to configure it. First, we have to create a User model that inherits from IdentityUser, for example:
```

Comment:
Then, we need to configure it. First, we have to create a User model that inherits from IdentityUser, for example: